src/azure-cli/setup.py

```diff
@@ -111,7 +111,7 @@
     'azure-mgmt-rdbms~=10.2.0b6',
     'azure-mgmt-recoveryservicesbackup~=5.1.0b1',
     'azure-mgmt-recoveryservices~=2.1.0',
-    'azure-mgmt-redhatopenshift==1.1.0',
+    'azure-mgmt-redhatopenshift==1.2.0',
```

Comment:
Usually, we will use `~=1.2.0` instead of `==1.2.0` in `setup.py` file for SDK version.

Reply:
This has been changed.